packages/sit-onyx/src/components/examples/GridPlayground/GridOverlay/GridOverlay.vue

```diff
@@ -40,6 +49,37 @@ const props = withDefaults(
 
   &__column {
     background-color: var(--onyx-color-text-icons-warning-soft);
+    position: relative;
+  }
+
+  &__grid-lines {
+    &::before {
+      content: " ";
+      display: inline-block;
+      position: absolute;
+      // I've set a precise height, because the 100% was getting the height of the
+      // overlay and was not overflowing till the bottom.
+      // I would like to hear suggestions for a more optimized value for the height
+      height: 31.25rem;
+      z-index: 60;
+      border: var(--onyx-1px-in-rem) dashed var(--onyx-color-component-border-danger-hover);
+    }
+
+    &::after {
+      content: " ";
+      display: inline-block;
+      position: absolute;
+      height: 31.25rem;
+      z-index: 60;
+      right: -1px;
+      border: var(--onyx-1px-in-rem) dashed var(--onyx-color-component-border-danger-hover);
+    }
+  }
```

Comment:
We can use a large height, like `100000px` and use `overflow: hidden;` in the `GridPlayground.vue`:

```css
.playground {
  overflow: hidden;
  height: 100vh;
}
```

and here:

```suggestion
  &__grid-lines {
    &::before {
      content: " ";
      position: absolute;
      height: 10000px;
      z-index: 60;
      border: var(--onyx-1px-in-rem) dashed var(--onyx-color-component-border-danger-hover);
    }

    &::after {
      content: " ";
      position: absolute;
      height: 10000px;
      z-index: 60;
      right: 0;
      border: var(--onyx-1px-in-rem) dashed var(--onyx-color-component-border-danger-hover);
    }
  }
```